environments/prod-dev/.env.example

```
@@ -0,0 +1,16 @@
+# Cloudflare Configuration
+CLOUDFLARE_API_TOKEN=your_cloudflare_api_token_here
+CLOUDFLARE_TUNNEL_ID=your_cloudflare_tunnel_id_here
+
+# DcMaidBot Configuration
+BOT_TOKEN=your_telegram_bot_token_here
+OPENAI_API_KEY=your_openai_api_key_here
+NUDGE_SECRET=your_nudge_secret_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NUDGE_SECRET key should go before the OPENAI_API_KEY key

(UnorderedKey)